.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Python Tests
+
+on:
+  push:
+  pull_request:
```

Comment:
I think this is redundant since you cannot open a PR without pushing a commit (and additionally I think github counts opening a PR as push event). So I'd remove the `pull_request` trigger.

Reply:
ok, updated